File: src/main/kotlin/com/ks/culinario/data/entity/ShoppingListEntity.kt
package com.ks.culinario.data.entity

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "shopping_lists")
data class ShoppingListEntity(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Int? = null,
    val title: String = "",
    val description: String = "",
    val items: String = ""
)

File: dotnet/Culinario.Infrastructure/Persistence/Entities/ShoppingListEntity.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Culinario.Infrastructure.Persistence.Entities;

[Table("ShoppingLists")]
public class ShoppingListEntity
{
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int? Id { get; set; }

    public string Title { get; set; } = string.Empty;
    public string Description { get; set; } = string.Empty;

    public ICollection<ShoppingListItemEntity> Items { get; set; } = new List<ShoppingListItemEntity>();
}
